src/entities/purchase-params.ts

```diff
@@ -39,4 +40,16 @@ export interface PurchaseParams {
    * to the RC transaction as metadata.
    */
   metadata?: PurchaseMetadata;
+
+  /**
+   * Defines an optional override for the default branding appearance.
+   *
+   * This property is used internally at RevenueCat to handle dynamic themes such
+   * as the ones coming from the Web Paywall Links. We suggest to use the Dashboard
+   * configuration to set up the appearance since a configuration passed as parameter
+   * using this method might break in future releases of `purchases-js`.
+   *
+   * @internal
+   */
+  brandingAppearanceOverride?: BrandingAppearance;
```

Comment:
Do we need to update the API and export `BrandingAppearance`? We should also mark it as `@internal` in that case.

Reply:
I'm actually surprised our tests didn't catch the change in API actually... Not sure what happened..

Reply:
Weird, updating it.

Maybe is because we are not merging in main